eo-maven-plugin/src/main/java/org/eolang/maven/tojos/ForeignTojo.java

```diff
@@ -124,4 +193,64 @@ public ForeignTojo withDiscoveredAt(final Path path) {
         this.delegate.set(ForeignTojos.Attribute.DISCOVERED_AT.key(), path);
         return this;
     }
+
+    /**
+     * Set sodg.
+     * @param sodg Sodg.
+     * @return The tojo itself.
+     */
+    public ForeignTojo withSodg(final Path sodg) {
+        this.delegate.set(ForeignTojos.Attribute.SCOPE.key(), sodg.toString());
+        return this;
+    }
+
+    /**
+     * Set the xmir2.
+     * @param xmir The xmir2.
+     * @return The tojo itself.
+     */
+    public ForeignTojo withXmirSecond(final Path xmir) {
```

Comment:
@volodya-lombrozo Can we make this name simpler?  According to: https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html

Reply:
@Graur For now it's impossible, because [that](https://github.com/objectionary/eo/issues/1990)

Reply:
@volodya-lombrozo Yeah, I miss that